classes/ActionScheduler_WPCLI_QueueRunner.php

```diff
@@ -16,6 +16,9 @@ class ActionScheduler_WPCLI_QueueRunner extends ActionScheduler_Abstract_QueueRu
 	/** @var \cli\progress\Bar */
 	protected $progress_bar;
 
+	/** @var bool */
+	protected $print_timestamp = false;
```

Comment:
Similar to the `getCliTimestamp()` method, this variable is a misnomer. No timestamp is ever printed. It's a date/time string that's prepended, so the variable should be renamed `$print_date` or similar.

Reply:
Removed in 814b6b2df56baadaa721061a8b9cbafa182f1a1a.